modules/resource-group/output.tf

```diff
@@ -0,0 +1,14 @@
+output "id" {
+  description = "resource group id"
+  value       = var.provision ? ibm_resource_group.resource_group[0].id : data.ibm_resource_group.resource_group[0].id
+}
+
+output "default" {
```

Comment:
Do we need this outputs both default or provision

Reply:
Added as per cloud native toolkit resource group module ... if the GIS team needs it they can use them

https://github.com/cloud-native-toolkit/terraform-ibm-resource-group/blob/main/outputs.tf